### PR TITLE
Update docs for manual fluentd install/config

### PR DESCRIPTION
**Pull request by jeffreyc-splunk**

### CODE REVIEW

**Comment by jcheng-splunk** [comment]
cc @@tmorrisonsfx

File: docs/getting-started/windows-manual.md [hunk 1]
```diff
@@ -87,6 +88,48 @@ Start-Service splunk-otel-collector
...
+   section.  The Collector's default configuration file
+   (`\ProgramData\Splunk\OpenTelemetry Collector\agent_config.yaml`) listens
+   for log events on `127.0.0.1:8006` and sends them to the Splunk
+   Observability Cloud.
+1. Check [https://docs.fluentd.org/installation/install-by-msi#td-agent-v4](
+   https://docs.fluentd.org/installation/install-by-msi#td-agent-v4) to install
+   the Fluentd MSI.  Requires version 4.0 or newer.
+1. Configure Fluentd to collect log events and forward them to the Collector:
+   - Option 1: Update the default config file provided by the Fluentd MSI at
+     `\opt\td-agent\etc\td-agent\td-agent.conf` to collect the desired log
```
Comment (rmfitzpatrick) [inline docs/getting-started/windows-manual.md:107]: linux path?

File: docs/getting-started/windows-manual.md [hunk 2]
```diff
@@ -87,6 +88,48 @@ Start-Service splunk-otel-collector
...
+   - Option 2: The installed Collector package provides a custom Fluentd config
+     file
+     (`\Program Files\Splunk\OpenTelemetry Collector\fluentd\td-agent.conf`) to
+     collect log events from the Windows Event Log
+     (`\Program Files\Splunk\OpenTelemetry Collector\fluentd\conf.d\eventlog.conf`)
+     and forwards them to `127.0.0.1:8006`.  To utilize these files, backup any
+     files as necessary in the `\opt\td-agent\etc\td-agent\` directory, and
+     copy the contents from the
+     `\Program Files\Splunk\OpenTelemetry Collector\fluentd\` directory to the
+     `\opt\td-agent\etc\td-agent\` directory.
```
[the remark this answers is not in the document]
Comment (jcheng-splunk) [reply docs/getting-started/windows-manual.md:119]: It is the default path on windows: https://docs.fluentd.org/installation/install-by-msi#step-2-set-up-td-agent-conf

**Comment by tmorrisonsfx** [comment]
@jcheng-splunk Can you check the list line numbering in the #### Fluentd section of `docs/getting-started/linux-manual.md`?

**Comment by jcheng-splunk** [comment]
> @jcheng-splunk Can you check the list line numbering in the `Fluentd` sections of both docs? It looks like each item starts with number 1.

Markdown supports numbered lists with 1's and should render correctly so we don't have to keep track when changing the list (https://www.markdownguide.org/basic-syntax/#lists-1).  But I can change it to normal numbering for clarity sake.

**Pull request merged**